### PR TITLE
Added slide in/out css effect to sidebar

### PR DESCRIPTION
**Pull request by woubuc**
A simple css transition that slides the sidebar menu on mobile in and out, instead of just instantly showing/hiding it.

### CODE REVIEW

File: stylesheets/styles.css [hunk 1]
```diff
@@ -331,13 +337,6 @@ pr-badge a:focus::after  {
 }
 
 /* Min-width break to add more padding on wider screens */
-@media all and (max-width: 720px) {
-	#sidebar {
-		display: none;
-	}
-}
-
-/* Min-width break to add more padding on wider screens */
```
Comment (dakebl) [inline stylesheets/styles.css]: I'd still have something that hides the sidebar on mobile devices.

Comment (woubuc) [inline stylesheets/styles.css]: @dakebl By default it has the class "closed" in index.html, so it will start out hidden.

Comment (dakebl) [inline stylesheets/styles.css]: Do we not want the sidebar visible on larger screens?

Comment (woubuc) [inline stylesheets/styles.css]: Class `closed` is visible on larger screens, check [L358-361](https://github.com/woubuc/1pr/blob/f7c6754395a05d762de2718017f4afce2bb7432b/stylesheets/styles.css#L358-L361)

**Pull request merged**